### PR TITLE
Link DOM living spec

### PR DESCRIPTION
**Pull request by nielsdos**

### CODE REVIEW

File: entities/global.ent [hunk 1]
```diff
@@ -76,6 +76,10 @@
 <!ENTITY url.spec.dom3.document "http://www.w3.org/TR/2003/WD-DOM-Level-3-Core-20030226/DOM3-Core.html#core-i-Document">
 <!ENTITY url.spec.dom3.node "http://www.w3.org/TR/2003/WD-DOM-Level-3-Core-20030226/DOM3-Core.html#core-ID-1950641247">
 <!ENTITY url.spec.dom3.nodelist "http://www.w3.org/TR/2003/WD-DOM-Level-3-Core-20030226/DOM3-Core.html#core-ID-536297177">
+<!ENTITY url.spec.dom.living.node "https://dom.spec.whatwg.org/#interface-node">
+<!ENTITY url.spec.dom.living.attr "https://dom.spec.whatwg.org/#interface-attr">
+<!ENTITY url.spec.dom.living.characterdata "https://dom.spec.whatwg.org/#interface-characterdata">
+<!ENTITY url.spec.dom.living.comment "https://dom.spec.whatwg.org/#interface-comment">
```
Comment (Girgias) [inline entities/global.ent:82]: Maybe add a "base" entity which just has the URL? And have the others ones use that?

Comment (nielsdos) [reply entities/global.ent:82]: Hm maybe, we don't really do that for the rest though :shrug:

Comment (Girgias) [reply entities/global.ent:82]: We do it a lot in `language-snippets` but not here indeed...

Comment (nielsdos) [reply entities/global.ent:82]: Tomorrow problem...

Comment (nielsdos) [reply entities/global.ent]: Fixed

**Review by Girgias — APPROVED** [review]
Thank you!

**Pull request merged**